Offseason2022/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -135,6 +135,8 @@ public class Drivetrain extends SubsystemBase
   private boolean                           m_isQuickTurn         = false; // Quickturn mode active in curvature drive
   private boolean                           m_driveSlowMode       = false; // Slow drive mode active when climbing
   private double                            m_limelightDistance;
+  private double                            m_offset1;

```

Comment:
Better names would be m_offsetLeft and m_offsetRight, since that's what these are. And it will help others from misusing them if they don't decode 1 and 2 as left and right.